lib/utils/constant.dart

```diff
@@ -1 +1,6 @@
 const double hundred = 100;
+const String favorites = 'Favorites';
+const String homeIcon = 'assets/home.png';
```

Comment:
this can be removed now once you prefer to utilize the existing/built-in icons based on my feedback above 🙂